src/ci/shared.sh

```diff
@@ -15,6 +15,12 @@
 
 # See http://unix.stackexchange.com/questions/82598
 function retry {
+  # Only do retries on CI, locally they shouldn't really be necessary
+  if [ -z "$TRAVIS$APPVEYOR" ]; then
```

Comment:
An alternate to avoid unbound variable issues: `"${TRAVIS:-}${APPVEYOR:-}"`, though it's possible using `$CI` would be cleaner.